blocks/heading/heading.js

```diff
@@ -0,0 +1,60 @@
+function updateHeadingStructure(block) {
+  const headingBlock = block;
+  if (!headingBlock) return;
+
+  const children = [...headingBlock.children];
+  if (children.length < 7) return;
```

Comment:
If optional fields are not authored, will this still be 7?

Reply:
It was breaking earlier, but its working now with the content thats authored. So removing the check.